src/gui/update.rs

```diff
@@ -85,25 +85,6 @@ pub fn handle_message(ajour: &mut Ajour, message: Message) -> Result<Command<Mes
                         }
                     });
 
-                    ajour.catalog_header_state.columns.iter_mut().for_each(|a| {
-                        if let Some((idx, column)) = columns
-                            .iter()
-                            .enumerate()
-                            .filter_map(|(idx, column)| {
-                                if column.key == a.key.as_string() {
-                                    Some((idx, column))
-                                } else {
-                                    None
-                                }
-                            })
-                            .next()
-                        {
-                            a.width = column.width.map_or(Length::Fill, Length::Units);
-                            a.hidden = column.hidden;
-                            a.order = idx;
-                        }
-                    });
```

Comment:
`ColumnConfig::V2` is from before we implemented the catalog, so this config only relates to the MyAddons stuff